src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int LIMIT = 70;
+    private static final int DEFAULT_RADIUS = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int numberFigure = random.nextInt(7);
+        switch (numberFigure) {
+            case 1:
+                return prepareSquare();
+            case 2:
+                return prepareRectangle();
+            case 3:
+                return prepareCircle();
+            case 4:
+                return prepareIsoscelesTrapezoid();
+            case 5:
+                return prepareRightTriangle();
+            default:
+                return getDefaultFigure();

```

Comment:
better throw an exception, here it's not intended behavior, we should always be in the valid range